game/graphics/pipelines/opengl.cpp

```diff
@@ -436,6 +439,15 @@ static void gl_render_display(GfxDisplay* display) {
     glfwSwapInterval(req_vsync);
   }
 
+  //Turn on/off msaa, if requested.
+  bool req_msaa = g_gfx_data->debug_gui.get_msaa_flag();
+  if(req_msaa != g_gfx_data->msaa_enabled){
+    g_gfx_data->msaa_enabled = req_msaa;
+    //Turn on multisampling if requested by user, turn off otherwise.
+    if(req_msaa) glEnable(GL_MULTISAMPLE);
```

Comment:
maybe we should call `glEnable(GL_MULTISAMPLE)` somewhere during startup just in case - I don't know what the default value will be and nothing seems to set it now. It seems like the sort of thing that might be different on different graphics drivers.

Reply:
It looks like GL_MULTISAMPLED is enabled by default, so there's probably no need to enable it on startup.

https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glEnable.xhtml
![image](https://user-images.githubusercontent.com/16104528/154383588-b335575c-0d20-4cd6-8bdf-ce86e651f9cb.png)

Thanks for the feedback! I'll get the formatting done.